node/cmd/guardiand/adminserver.go

```diff
@@ -388,3 +390,9 @@ func adminServiceRunnable(logger *zap.Logger, socketPath string, injectC chan<-
 	publicrpcv1.RegisterPublicRPCServiceServer(grpcServer, publicrpcService)
 	return supervisor.GRPCServer(grpcServer, l, false), nil
 }
+
+func (s *nodePrivilegedService) SendObservationRequest(ctx context.Context, req *nodev1.SendObservationRequestRequest) (*nodev1.SendObservationRequestResponse, error) {
+	s.obsvReqSendC <- req.ObservationRequest
+	s.logger.Info("sent observation request", zap.Any("request", req.ObservationRequest))
+	return &nodev1.SendObservationRequestResponse{}, nil
```

Comment:
Forgive my ignorance, but why return an empty `*nodev1.SendObservationRequestResponse`?

Reply:
The `SendObservationRequestResponse` message is just an empty placeholder:

```proto
service NodePrivilegedService {
  // ...
  rpc SendObservationRequest (SendObservationRequestRequest) returns (SendObservationRequestResponse);
}
message SendObservationRequestRequest {
  gossip.v1.ObservationRequest observation_request = 1;
}

message SendObservationRequestResponse {}
```

This is best practice for gRPC, since it allows us to add fields to the response later without breaking backwards compatibility.

Reply:
Ah forgive my ignorance here, this is my first non-trivial grpc project. I'm learning on this stuff.